apisix/core/pubsub.lua

```diff
@@ -119,8 +138,8 @@ end
 -- no error exists.
 --
 -- @function core.pubsub.on
--- @tparam string command to add callback
--- @tparam function handler callback on receipt of command
+-- @tparam string command The command to add callback.
+-- @tparam func handler The callback function on receipt of command.
```

Comment:
`-- @tparam function` ?

Reply:
![image](https://user-images.githubusercontent.com/8078418/168267423-0fe75a46-7831-4109-bb47-7e12d711e7de.png)

The documentation indicates that `func` should be used.